sql/updates/world/3.3.5/2017_09_25_99_world.sql

```diff
@@ -0,0 +1,21 @@
+--
+DELETE FROM `gossip_menu` WHERE `MenuID` BETWEEN 57020 AND 57022;
+INSERT INTO `gossip_menu` (`MenuID`, `TextID`, `VerifiedBuild`) VALUES
+(57020,9227,0),(57021,9229,0),(57022,9231,0);
```

Comment:
This gossips are already on the DB, just use them, no need to create new ones:
 7583, 9227
 7585, 9229
 7586, 9231

Reply:
Oh. Right. I forgot to look up those.
```sql
SELECT * FROM `gossip_menu` WHERE `TextID` IN (9227,9229,9231);
```
```
MenuID  TextID  VerifiedBuild
------  ------  -------------
  7583    9227              0
  7585    9229              0
  7586    9231              0
```

Reply:
rusty I guess, my bad!